src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -20,6 +20,8 @@
 #include "ColorHelper.h"
 #include "DebugTapConnection.h"
 
+#include <algorithm>
```

Comment:
is this not already in `pch.h` or `LibraryIncludes.h` already?

Reply:
Removed, wasn't aware of those files.